ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py

```diff
@@ -13,8 +13,8 @@
 
 @st.composite
 def statistical_dtype_values(draw, *, function):
-    large_abs_safety_factor = 72
-    small_abs_safety_factor = 72
+    large_abs_safety_factor = 24
```

Comment:
Please can you revert them back to their original value which was `2`?
If it makes the test to fail only then do the change below I've mentioned.